examples/app/Embeddings.ipynb

```diff
@@ -20,36 +48,257 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "7619577f",
+   "metadata": {},
+   "source": [
+    "You can change the type of the displayed weave panel by changing `Table` to `Plot` and it will intelligently change to display a plot of the data.\n",
+    "![image.png](attachment:image.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "356dc2a9",
+   "metadata": {},
+   "source": [
+    "## Embed and Cluster the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fd47c57",
+   "metadata": {},
+   "source": [
+    "Now we can pass a column from this data to another weave `Op` to embed it. This will turn each row in the column `name` into a vector."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "43cec2d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "embeddings = openai.openai_embed(data['name'], {\"model\": \"text-embedding-ada-002\"})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6715de4a",
+   "metadata": {},
+   "source": [
+    "Now, we'll project this to 2 dimensions for clustering, and then we'll cluster this projection using `hdbscan`."
```

Comment:
minor: I think it's ok to generally assume folks know what you're talking about/not block on this, and I would love an extra sentence or note or two on what is happening and why—or recommendations for which parameters to play with and why ("try modifying the number of neighbors, or the min dist, to get at these properties/explore the clustering in this way").

Reply:
Yeah, agreed some explanation would be good here. I actually tried to find some but there wasn't a lot.